nestjs/src/courses/course-schedule/course-schedule.service.ts

```diff
@@ -304,7 +304,7 @@ export class CourseScheduleService {
         ...(technicalScreeningResults
           .find(task => task.courseTaskId === courseTaskId)
           ?.stageInterviewFeedbacks.map(feedback => JSON.parse(feedback.json))
-          .map((json: any) => json?.resume?.score ?? 0) ?? []),
+          .map((json: any) => (json?.resume?.score || json?.steps?.decision?.values?.finalScore) ?? 0) ?? []),
```

Comment:
Let's split it a bit and make more readable  
```suggestion
          .map((json: any) => {
    const resumeScore = json?.resume?.score;
    const decisionScore = json?.steps?.decision?.values?.finalScore;
    return resumeScore ?? decisionScore ?? 0;
});
```